bot/exts/evergreen/tic_tac_toe.py

```diff
@@ -320,7 +320,7 @@ async def show_tic_tac_toe_board(self, ctx: Context, game_id: int) -> None:
 
         embed = discord.Embed(
             title=f"Match #{game_id} Game Board",
-            description=f"{game.winner} :trophy: vs {game.loser}\n\n{game.format_board()}"
+            description=f"{game.winner} vs {game.loser} (draw)\n\n{game.format_board()}" if game.draw else f"{game.winner} :trophy: vs {game.loser}\n\n{game.format_board()}"
```

Comment:
Thanks for the contribution!

As stated in the linting error, this line is a bit too long because of the ternary, perhaps we can shorten it by extracting `description` into a variable.

```py
if game.draw:
    description = f"{game.winner} vs {game.loser} (draw)\n\n{game.format_board()}"
else:
    description = f"{game.winner} :trophy: vs {game.loser}\n\n{game.format_board()}"
```

Reply:
Thanks for the help. I've made the commits removing the ternary operators.